README.md

```diff
@@ -34,6 +34,18 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+## Setup Trello
```

Comment:
Commenting here because the relevant files haven't change, but when I tried to run the app I got complaints that the `requests` module wasn't installed - this may just mean you've not added/committed any changes to the `pyproject.toml` and `poetry.lock` files, or may be a sign that you're running the app through a separate command that isn't relying on the project-specified dependencies which can lead to different developers having different experiences, so best to:
* Add the `requests` library once at your terminal with `poetry add requests`
* Check in the changes to `pyproject.toml` and `poetry.lock` so that future developers get the same libraries/versions when they run `poetry install`
* Make sure you're always running the app through poetry so that it's using the correct virtual environment (`poetry run flask run`)